include/swift/IRGen/Linking.h

```diff
@@ -110,6 +110,10 @@ class LinkEntity {
     /// A swift metaclass-stub reference.  The pointer is a ClassDecl*.
     SwiftMetaclassStub,
 
+    /// A class metadata base offset global variable.
+    /// The pointer is a ClassDecl*.
+    ClassMetadataBaseOffset,
```

Comment:
I'm not really sure where the right place is, but somewhere you should actually describe what this means.  I think it's the offset to the start of the class-specific section of the class metadata?

Reply:
That is correct. I'll improve the comment here, thanks!